charts/woodpecker/README.md

```diff
@@ -16,6 +16,13 @@ helm install woodpecker woodpecker/woodpecker
 **Note**: The `woodpecker/woodpecker` chart contains both the server and the agent.
 If you want to install only the server or agent you can use the [`woodpecker/server`](./charts/server/README.md) or [`woodpecker/agent`](./charts/server/README.md) charts respectively.
 
+### Secrets
+
+Since version 1.1.0 the chart bootstrap a secret for `WOODPECKER_AGENT_TOKEN` which is required for the communication between agent and server.
+You might want to add additional secrets for other sensitive settings like the client secret of your respective forge provider or `WOODPECKER_DATABASE_DATASOURCE`.
```

Comment:
So how would I add and provide such a secret?

Reply:
Create a secret outside of the chart and reference it within `secrets:` so it is available in the deployment.

Reply:
Is that mentioned somewhere so we can link to it or should we add an example.

Reply:
I'd say that this is "common knowledge" when working with helm charts. You don't want to define secrets in plain text obviously but create them manually/outside and then inject them into the chart.

The config hasn't changed actually, users were also able to insert secrets via `secrets:` previously, we only generate the agent token now internally while beforehand it was empty/just a placeholder (due to multiple requests from users that this should be handled internally). 

What we could/should do in addition is tell users that all sensitive configs should be provided like this, as otherwise they are exposed in plain text in the pod config when using env vars.

Reply:
```suggestion
You might want to add additional secrets for other sensitive settings like the client secret of your respective forge provider or `WOODPECKER_DATABASE_DATASOURCE`. Therefore you would create a secret `woodpecker-secret` containing: `WOODPECKER_DATABASE_DATASOURCE` and add it as `secrets: [{"name":"woodpecker-secret"}]`
```

Reply:
I wouldn't add a name for the secret as otherwise some users will think that it "must be named that way". Again, I am not sure if we should try to explain k8s basics in the chart README (?). 

Can you maybe explain more what confuses you without that specific detail?